pkgs/development/compilers/rust/rustc.nix

```diff
@@ -56,7 +56,18 @@ stdenv.mkDerivation {
 
   patches = patches ++ targetPatches;
 
-  passthru.target = target;
+  passthru = {
+    unpackedSrc = stdenv.mkDerivation {
+      name = "rust-src-${version}";
+      inherit src;
+      installPhase = ''
+        mkdir -p $out
+        cp -r . $out/
+      '';
+      phases = [ "unpackPhase" "installPhase" ];
```

Comment:
Various people don't like setting phases like this. You can skip a phase by setting it to ":" (a command which does the same as the command true).